homeworks/kirkovvalentiniván/2_vars_datatypes/user_info.py

```diff
@@ -0,0 +1,46 @@
+from pprint import pprint
+
+user_info = {
+    "name": "Mike",
+    "age": 25,
+    "favourite_meals": [
+        "pizza",
+        "carbonara",
+        "sushi"
+    ],
+    "phone_contacts": {
+        "Mary": "+36701234567",
+        "Tim": "+36207654321",
+        "Tim2": "+36304567321",
+        "Jim": "+364005000"
+    }
+}
+
+prog_language = input("List 4 programming languages. Separate them by comma without using space.")  #1a Ask for 4 programming languages
+prog_language_list = prog_language.split(",")                                                       #1b Conversion to lists
+user_info["skills"] = prog_language_list                                                            #1c List adding to dictionaries
+
+user_info["favourite_meals"] = sorted(user_info["favourite_meals"])                                 #2. Sorting to alphabetical order
+
+print(user_info["favourite_meals"][-2])                                                             #3. Printing the penultimate element
+
+user_info["favourite_meals"] += ["spaghetti"]                                                       #4. Adding spaghetti to the original list
+
+user_info["favourite_meals"] += user_info["favourite_meals"][2:4]                                   #5. Appending the 3rd and 4th element of the list to favourite_meals 
```

Comment:
ennél, és az előzőnél az .append() és az .extend() használatára lettem volna kiváncsi, de ez is tökéletes megoldás :) 
szép munka, hogy list slicing-ot használtál, és nem külön külön adtad hozzá a 3. és 4. elemet